jobs/integr8ly/pds-general.yaml

```diff
@@ -107,6 +113,17 @@ This needs to be changed to 'false' for Integreatly Workshop since it uses valid
                     
                     stage('Install #1') {
                         if(TO_DO.contains('heavy') || TO_DO.contains('install')) {
+
+                            if (INSTALLATION_BRANCH == 'master') {   
+                                // patch all the integreatly components to use master tag version
+                                // first remove the line containing the variable and then insert a new one
+                                for(String tag in intlyCompTags){
+                                    sh """
+                                        sed -i "/${tag}: / d" evals/inventories/group_vars/all/manifest.yaml
+                                        sed -i "1 a ${tag}: 'master'" evals/inventories/group_vars/all/manifest.yaml
+                                    """
+                                }
+                            }
```

Comment:
This doesn't work. We need to patch the manifest directly in installation-pipeline